.github/workflows/test-flux.yaml

```diff
@@ -1,9 +1,78 @@
-name: Test Flux Executor
+name: Test Flux Executors
 on:
   pull_request: []
 
 jobs:
-  build:
+  test-flux-operator-executor:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # Note that we cannot test conda because the bind of the filesystem is too slow
+        commands: [["singularity", "cp -R ./tests/test_singularity_module/* /tmp/workflow/", "snakemake --cores 1 --jobs 2 --flux-operator --use-singularity"],
+                   ["vanilla", "", "snakemake --cores 1 --jobs 1 --flux-operator --minicluster-container-image ghcr.io/rse-ops/atacseq-vanilla:app-latest"]]
```

Comment:
I think the option should be called --flux-minicluster-container-image. And can you remind me again in what sense this differs from the ordinary --container-image option? Do we really need a separate one?

Reply:
I only created a separate one because the first container image was in a different argument group. Do you have a suggested way of implementing this?

Reply:
Just move the --container-image argument into group_exec and adapt the help text of --container-image.